harness/tests/experiment/fixtures/ds_config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
why is this here, but also copied into the test_deepspeed_trial.py file?

Reply:
i removed the dictionary and it's now reading from the json file.